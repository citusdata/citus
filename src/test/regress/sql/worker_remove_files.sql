-- Clear job directory used by previous tests

\set JobId 201010

SELECT task_tracker_cleanup_job(:JobId);
