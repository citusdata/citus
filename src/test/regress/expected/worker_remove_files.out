-- Clear job directory used by previous tests
\set JobId 201010
SELECT task_tracker_cleanup_job(:JobId);
 task_tracker_cleanup_job
---------------------------------------------------------------------

(1 row)

